src/components/Wizards/CreateVmWizard/tests/CreateVmWizard.test.js

```diff
@@ -126,6 +127,23 @@ describe('<CreateVmWizard />', () => {
     expect(component.find(WizardPattern).props().nextStepDisabled).toBeTruthy();
   });
 
+  it('first step renders <BasicSettinsTab />', () => {
```

Comment:
there is a mistake at https://github.com/kubevirt/web-ui-components/pull/51/commits/a449f3935fea5c89e884da1a64c5e0db21091534#diff-1007c24f3f684e98ae9b3e620fd4c85eR43: this isn't merged yet and shouldn't belong to `e09e902 Add networks tab and PXE boot (#41)` 

Please could you try to incorporate these two use cases into the testWalkThrough method if it is feasible?